userguide/system.rst

```diff
@@ -432,6 +434,9 @@ The buttons above the boot entries can be used to:
   :numref:`Figure %s <status_boot_dev_fig>`,
   shows only one boot device, which is *ONLINE*.
 
+.. note:: Using :guilabel:`Clone` to clone the active boot environment
```

Comment:
This is not exactly correct.  If it functions the same, we would not need two separate buttons.

Please expand on this.  I suspect it asks for a name, then clones a boot environment (only the active one?).

Reply:
Using "Clone" snaps the selected environment. (this includes selecting and cloning the active environment)
Using "Create" snaps the current "Active" environment.

Reply:
UPDATE: I've talked to Caleb about the functionality and to clarify things about it.
Essentially The "Create" and "Clone" button DO the same thing. However, the only difference is that if a user uses "Create" it makes a new boot environment based on the "Active" environment. The "Clone" button makes a new boot environment based on the boot environment selected.

I will put in a ticket.